__snapshots__/index.test.js.snap

```diff
@@ -2757,6 +2805,12 @@ Array [
 
 exports[`should pass on packages/eslint-config-vue/test/rules/vue-no-shared-component-data.vue 1`] = `
 Array [
+  Object {
+    "column": 1,
+    "line": 5,
+    "rule": "import/newline-after-import",
```

Comment:
Can you fix this error by fixing `packages/eslint-config-vue/test/rules/vue-no-shared-component-data.vue`?